tests/test_unauthenticated_reddit.py

```diff
@@ -211,8 +211,18 @@ def test_get_top(self):
 
     @betamax()
     def test_info_by_id(self):
-        self.assertEqual(self.link_id,
-                         self.r.get_info(thing_id=self.link_id).fullname)
+        # individual
+        submission = self.r.get_info(thing_id=self.link_id)
+        self.assertEqual(self.link_id, submission.fullname)
+
+        # groups
```

Comment:
Can we make the group a separate test so that we can ensure the original cassette still works without modification?